ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala

```diff
@@ -209,10 +209,22 @@ class ProcessValidation(modelData: ProcessingTypeDataProvider[ModelData],
   }
 
   private def validateWithCustomProcessValidator(process: DisplayableProcess): ValidationResult = {
-    customProcessNodesValidators
+
+    val errors = customProcessNodesValidators
       .forType(process.processingType)
-      .map(_.validate(process))
-      .getOrElse(ValidationResult.success)
+      .map(_.validate(ProcessConverter.fromDisplayable(process)))
+      .getOrElse(List())
+    val globalErrors = errors.filter(_.nodeIds.isEmpty)
```

Comment:
1. let's just use formatErrors?
2. let's move the stuff into validateUsingTypeValidator, so that we don't handle model/canonicalProcess conversion twice?